src/ui/settings/colorMode.js

```diff
@@ -0,0 +1,27 @@
+//; -*- mode: rjsx;-*-
+/** @jsx jsx */
+
+import { jsx, useColorMode } from 'theme-ui';
+import { Button } from '@theme-ui/components';
+import { useTranslation } from 'react-i18next';
+
+import { SettingsSection } from './elements';
+
+const ColorModeSettings = () => {
+  const { t } = useTranslation();
+  const [colorMode, setColorMode] = useColorMode();
+
+  const dark = t('settings-theme-dark');
+  const light = t('settings-theme-light');
+
+  return (
+    <SettingsSection title={t('settings-theme-appearance')}>
+      <Button
+        onClick={() => setColorMode(colorMode === 'dark' ? 'light' : 'dark')}>
+        {colorMode === 'dark' ? light + ' ☀️' : dark + ' 🌙'}
```

Comment:
I like these emojis, but unfortunately they render very differently on different devices and operating systems. This has caused trouble in the past. So I think it would be better to use font awesome icons here. (They are used across studio)